terraform-aws-zhe-backend/README.md

```diff
@@ -0,0 +1,164 @@
+  <img alt="ZenHub" src="terrafom-aws-zhe-backend.png" width="1000" />
+</div>
+
+## Table of Contents
+
+1. [Usage](#Usage)
+2. [Subnet calculation logic](#subnet-calculation-logic)
+3. [Requirements](#Requirements)
+4. [Providers](#Providers)
+5. [Inputs](#Inputs)
+6. [Outputs](#Outputs)
+7. [Authors](#authors)
+
+---
+
+## Usage
+
+```hcl
+module "zhe" {
+  source              = "source = "github.com/zenhubHQ/zenhub-enterprise/terraform-aws-zhe-backend"
+  env    = "test"
+  region = "us-west-2"
+  vpc_id = "vpc-0000aaaa11b2cc111"
+
+  postgres_vars = {
+    instance_class = "db.t3.small"
+  }
+
+  redis_vars = {
+    node_type = "cache.t3.small"
+  }
+
+  documentdb_vars = {
+    instance_class = "db.t3.medium"
+    instance_count = "1"
+  }
+}
+```
+
+---
+
+## Subnet calculation logic
+
+`terraform-aws-zhe-data` creates a set of subnets based on `var.cidr_netnum`, the existing `VPC CIDR` and number of Availability Zones in the region.
+
+For subnet set calculation, the module uses Terraform interpolation [cidrsubnet](https://www.terraform.io/docs/configuration/functions/cidrsubnet.html).
+
+`cidrsubnet(iprange, newbits, netnum) - Takes an IP address range in CIDR notation (like 10.0.0.0/16) and extends its prefix to include an additional subnet number. For example, cidrsubnet("10.0.0.0/16", 8, 100) returns 10.0.100.0/24 and ("10.0.0.0/16", 8, 101) returns 10.0.101.0/24;`
+
+`newbits` is the number of additional bits with which to extend the prefix. For example, if given a prefix ending in /16 and a newbits value of 4, the resulting subnet address will have length /20.
+
+`netnum` is a whole number that can be represented as a binary integer with no more than newbits binary digits, which will be used to populate the additional bits added to the prefix.
+
+```hcl
+cidrsubnet(data.aws_vpc.zhe.cidr_block, 8, count.index + var.cidr_netnum)
+```
+
+* `data.aws_vpc.zhe.cidr_block` = CIDR of the existing VPC
+* `count.index` = sequencial id of subnets to be created
+* `var.cidr_netnum` = input variable
+* `count.index + var.cidr_netnum` = final `netnum` to return new subnet CIDR
+
+---
+
+## Requirements
+
+| Name | Version |
+|------|---------|
+| terraform | >= 0.13.5 |
+| aws | >= 3.18 |
+
+## Providers
+
+| Name | Version |
+|------|---------|
+| aws | >= 3.18 |
+| random | >= 3.0 |
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| region | AWSregion | `string` | `[]` | YES |
+| vpc_id | VPCidtocreatesubnetsanddatabases | `string` | `[]` | YES |
+| env | Tag to identify zenhub enviroment and resources | `string` | `` | NO |
+| creator | Tag to identiy who created this module's resources | `string` | `Terraform` | NO |
+| bucket_force_destroy | A boolean that indicates all objects (including any locked objects) should be deleted from the bucket so that the bucket can be destroyed without error. These objects are not recoverable. | `bool` | `true` | NO |
```

Comment:
```suggestion
| bucket_force_destroy | A boolean that indicates all objects (including any locked objects) which should be deleted from the bucket so that the bucket can be destroyed without error. These objects are not recoverable. | `bool` | `true` | NO |
```